x-pack/test/functional/apps/ml/data_frame_analytics/classification_creation.ts

```diff
@@ -43,24 +42,21 @@ export default function ({ getService }: FtrProviderContext) {
         createIndexPattern: true,
         expected: {
           rocCurveColorState: [
-            // background
-            { key: '#FFFFFF', value: 93 },
             // tick/grid/axis
-            { key: '#98A2B3', value: 1 },
-            { key: '#DDDDDD', value: 3 },
-            // line
-            { key: '#6092C0', value: 1 },
+            { key: '#DDDDDD', value: 48 },
```

Comment:
Running locally on Windows, I had to edit these values to the following to get it to pass:

```
            // tick/grid/axis
            { key: '#DDDDDD', value: 50 },
            // lines
            { key: '#98A2B3', value: 30 },
            { key: '#6092C0', value: 10 },
            { key: '#5F92C0', value: 6 },
```